pkgs/by-name/wl/wl-gammarelay-applet/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  stdenv,
+  wayland,
+  libxkbcommon,
+  fontconfig,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wl-gammarelay-applet";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "lgbishop";
+    repo = "wl-gammarelay-applet";
+    rev = "8a0d9e6364d7445fc69c59b2f168cfec91c2fe87";
+    sha256 = "sha256-t6bycmaquZ0IMs/WnAzkz5FnIWKIq0BTbeeoUFLeuYg=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
```

Comment:
Is their a particular reason to add `Cargo.lock`? Due to the current issues with repo size and eval overhead, we tend to use `cargoHash`, although this strategy may change in the future.

Reply:
No reason in particular - have changed it: [https://github.com/lgbishop/nixpkgs/commit/9b2a6631388462930c9a481512c45829c4c0f117](url)